Sito/js/funzioni.js

```diff
@@ -76,7 +76,7 @@ function checkNomeCognome(input) {
     }
 }
 function checkSoloLettereEDim(input) {
-    var patt = new RegExp('^[a-zA-Z]{2,}$');
```

Comment:
Ma sei sicuro che questo non si usato in un posto dove non si vogliono spazi? Ad esempio io credo di averlo usato nel nome utente